GVFS/GVFS.Hooks/HooksPlatform/GVFSHooksPlatform.Mac.cs

```diff
@@ -1,42 +1,42 @@
-using GVFS.Platform.Mac;
+using GVFS.Platform.POSIX;

```

Comment:
Not sure if we have established a hard rule but using `Posix` instead of `POSIX` reads better to me.  What are you thoughts @wilbaker

Reply:
I agree, I prefer `Posix`

Reply:
I'll be the dissenting opinion here. We have precedent for not camel-casing acronyms in GVFS. We should probably stick with POSIX for the same reason.

Reply:
> We have precedent for not camel-casing acronyms in GVFS. We should probably stick with POSIX for the same reason.

That's a good point, we use `GVFS` in the code rather than `Gvfs`.  I'm good with sticking with `POSIX`.